src/stores/user-store.js

```diff
@@ -74,6 +74,17 @@ class UserStore {
     return value;
   }
 
+  async updateUserUsedBytes({ userId, usedBytes }, { session } = {}) {
+    const filter = { _id: userId };
+    const update = { $set: { 'storage.usedBytes': usedBytes } };
+    const options = { session, returnDocument: 'after' };
+
+    const { value } = await this.collection.findOneAndUpdate(filter, update, options);
+
+    validate(value, userDBSchema);
```

Comment:
I presume this is a leftover, otherwise I'm not sure why we should validate a value returned from the database against our expected schema, since the DB data is already updated at this point.